docs/LanguageGuide.md

```diff
@@ -69,6 +69,58 @@ foo ; symbol
 (defmodule <name> <definition1> <definition2> ...) ;; The main way to organize your program into smaller parts
 ```
 
+### Conditional statements with 'cond'
+```(doc cond "this is the documentation for cond")```
```

Comment:
Not sure what purpose this line has....

Reply:
I think this was because of my comment above demonstrating the usage of `doc`. I should’ve been more clear about this: That line goes into the file where `cond` is defined, and the docstring shouldn’t be literally what I wrote above, but an actual documentation for `cond`. Sorry for the confusion, @eriksvedang and @mitchell-frost!